Formula/n/notation.rb

```diff
@@ -0,0 +1,32 @@
+class Notation < Formula
+  desc "CLI tool to sign and verify OCI artifacts and container images"
+  homepage "https://notaryproject.dev/"
+  url "https://github.com/notaryproject/notation.git",
+    tag:      "v1.0.0",
+    revision: "80e3fc4e2eeb43ac00bc888cf41101f5c56f1535"
```

Comment:
I'm wondering what's the difference between this one and
```
  url "https://github.com/notaryproject/notation/archive/refs/tags/v1.0.0.tar.gz"
  sha256 "c7e3a154cb132d17c72c9954e2c2144b5dd1781af69e9bfe25ec29916e2f01be"
```

Reply:
So I used the git method to add the GitCommit in the Build Flags, to use `#{Utils.git_short_head}`

Reply:
yeah, that is the catch, I would also see you can use `#{tap.user}` to fill that commit, the commit is more relevant to upstream build, not to homebrew build though.